packages/styles/components/_c.text-input.scss

```diff
@@ -13,11 +11,6 @@
   background-color: $color-input-background;
   border-radius: get-border-radius('m');
 
-  // remove border height and center baseline optically
-  padding:
-    calc(#{$mu075} - 0.125em - #{$border-width}) $mu075
-    calc(#{$mu075} + 0.125em - #{$border-width});
-
   &::placeholder {
     @include set-font-scale('05');
```

Comment:
Put the placeholder inside your mixin and define the same font-size and line-height for it as the value. 
on safari this is what it look like :
<img width="432" alt="Capture d’écran 2019-12-04 à 18 14 10" src="https://user-images.githubusercontent.com/14542464/70164692-ec77e100-16c1-11ea-8966-96070ba9ae37.png">

Reply:
If it is not the source of the problem, I will pull your branch to deboug this.

Reply:
I make the modification, but I can't test it, 'cause it's a Safari issue.
Can you check on your side and tell me if it's okay?

Reply:
@gael-boyenval about the line-height, it's fixed, but about the icon-size the designers decided to leave it like that.